### PR TITLE
Added template node.

### PR DESCRIPTION
**Pull request by robertsj**
As title says.

### CODE REVIEW

File: template-node/exercises/special-relativity-example.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+uid: special-relativity-example
+title: Special Relativity Example Problem
+type: MultipleChoice
+options:
+  text: Who is known for $E=mc^2$?
+  correctAnswers: 
+    - Einstein
+  incorrectAnswers:
```
Comment (davis68) [inline template-node/exercises/special-relativity-example.yml]: I propose that we support the ability to comment on incorrect answers.  If we are using distractor answers in MC questions, notation on why the answers are distractors is potentially critical.

**Comment by davis68** [comment]
I propose that we support the ability to comment on incorrect answers. If we are using distractor answers in MC questions, notation on why the answers are distractors is potentially critical.  (I.e., which failure of understanding they are testing against.)

**Comment by sskutnik** [comment]
I agree with @davis68; coming up with useful distractors is often a very good diagnostic tool and can be especially useful for constructing automated assessments (e.g., online quizzes).

**Review by sskutnik — APPROVED** [review]
I like this example with (optional) answer comments.

Naive question - how (or will) these render?

**Pull request merged**